Movecraft/src/main/java/net/countercraft/movecraft/async/AsyncManager.java

```diff
@@ -293,7 +293,7 @@ private void processSinking() {
             if (!(craft instanceof SinkingCraft))
                 continue;
 
-            if (craft.getHitBox().isEmpty() || craft.getHitBox().getMinY() < 5) {
+            if (craft.getHitBox().isEmpty() || craft.getHitBox().getMinY() < (craft.getWorld().getMinHeight() +5 )) {
```

Comment:
Just a thought but is there really any reason we even check for it being 5 over the min in the firs place 🤔

Reply:
I removed it on my fork and everything works fine. I kept it here because I didn't know if you have any other classes that need it. I k ow the IWorldHandler has a Y>>4 
https://github.com/APDevTeam/Movecraft/blob/6422b9857e52071fa28ad476b5ce9dfd64b6c67d/v1_20/src/main/java/net/countercraft/movecraft/compat/v1_20/IWorldHandler.java#L253
which may be the original reason why you had this set to y<5.